src/storage/store.js

```diff
@@ -96,7 +96,6 @@ export const SCHEMA = {
         preferredMic: { type: "string" },
         preferredCamera: { type: "string" },
         muteMicOnEntry: { type: "bool" },
-        audioOutputMode: { type: "string" },
```

Comment:
If you remove this key from the store, but it was set in a user's local storage because in their most recent setting they set their `audioOutputMode` pref (via the command), then when the user loads the page, the data in their local storage will cause them to fail the schema validation and the page will fail to load.

Before schema validation runs the first time, we need to delete this entry from users' local storage. Alternatively, we can leave it in the schema definition for a while (with code anywhere that deletes the stored value) with a comment that says, "Deprecated, to be removed once we're confident local storage has been cleaned up."

I think I like the first option better, where before doing schema validation we delete now-deprecated keys if they exist.

Reply:
I think we switched the preferences store to accept additional properties here:
https://github.com/mozilla/hubs/pull/4275/commits/e865eb78812980297b6ce6fae5236e3b737b0911#r636548037
That's why it doesn't really crash (and also why it doesn't crash even though I forgot to add the actual new `disableLeftRightPanning` pref 😬) I wonder if we should write cleanup code though so old prefs are removed and the store doesn't hold stall prefs.

Reply:
Oh, great!